docs/pages/router/advanced/customizing-links.mdx

```diff
@@ -0,0 +1,153 @@
+---
+title: Customizing links
+description: Learn how to perform link redirection and working with 3rd-party deep-links
+---
+
+import { FileTree } from '~/ui/components/FileTree';
+import { APIBox } from '~/components/plugins/APIBox';
+
+Expo Router offers a adaptable solution for managing navigation within your Expo app. However, its URL handling philosophy may not align with every use case or integration with third-party services. There are two common scenarios where you may need to customize a link:
+
+- **App Closed**: When the app is not open, incoming deep-link URLs may require rewriting to ensure seamless navigation.
+
+- **App Open**: In instances where the app is already open, URL customization may be necessary based on specific business logic or user interactions. This logic could be global for the entire application, or localized to a set of routes.
+
+## Setting up linking
+
+Please refer to the [Linking docs](https://docs.expo.dev/guides/linking/) on instructions on how to setup and test linking to your app.
```

Comment:
```suggestion
Please refer to the [Linking docs](/guides/linking/) on instructions on how to setup and test linking to your app.
```

We prefer using internal links, with external link notation we trigger full app reload on navigation.